docs/submitting_data.rst

```diff
@@ -168,7 +168,7 @@ data structures to send, do something like this.
 
 .. code-block:: python
 
-    from thclient import (TreeherderAuth, TreeherderClient, TreeherderClientError,
+    from thclient import (HawkAuth, TreeherderClient, TreeherderClientError,
```

Comment:
Eugh, right, I forgot about hawk_requests. On a train so I can't test stuff so easily.

Anyway, I like both of these ideas.

If we do the former (add a TreeherderHawkAuth class, or something like it), let's wait to update the docs until that lands.